src/components/courseSettings/ManageStaffPanel.tsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import {
+  Card,
+  CardBody,
+  CardHeader,
+  CardTitle,
+  ListGroup,
+  ListGroupItem,
+} from 'reactstrap'
+import FlipMove from 'react-flip-move'
+import AddStaff from '../AddStaff'
+
+import RemoveableUserItem from '../RemoveableUserItem'
+
+interface ManageStaffPanelProps {
+  course: {
+    id: number
+    name: string
+    staff: number[]
+  }
+  users: {
+    id: number
+    netid: string
+    name: string
+  }[]
+  removeCourseStaff: (courseId: number, userId: number) => void
+  addCourseStaff: (courseId: number, netid: string, name: string) => void
+}
+
+const ManageStaffPanel = ({
+  course,
+  users,
+  removeCourseStaff,
+  addCourseStaff,
+}: ManageStaffPanelProps) => {
+  let newUsers
+  if (course.staff && course.staff.length > 0) {
+    newUsers = course.staff.map(id => {
+      const user = users[id]
+      return (
+        <RemoveableUserItem
+          key={user.id}
+          onRemove={(userId: number) => removeCourseStaff(course.id, userId)}
+          {...user}
+        />
+      )
+    })
+  } else {
+    newUsers = (
+      <ListGroupItem className="text-center text-muted pt-4 pb-4">
+        This course doesn&apos;t have any staff yet
+      </ListGroupItem>
+    )
+  }
+
+  return (
+    <Card className="mb-3">
+      <CardHeader>
+        <CardTitle tag="h5" className="mb-0">
+          Manage {course && course.name} Staff
+        </CardTitle>
+      </CardHeader>
+      <CardBody>
+        <ListGroup flush className="position-relative">
+          <AddStaff
```

Comment:
`AddStaff` shouldn't have to be within the `ListGroup`. I'd switch this around to have just `AddStaff` directly inside the `CardBody` and put the `ListGroup` as a direct child of `Card` to get rid of some unnecessary padding.